tests/framework-tests/preferences/js/PanelTestUtils.js

```diff
@@ -29,6 +28,10 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         }
     });
 
+    fluid.defaults("fluid.tests.panels.utils.injectTemplates", {
+        listeners: {"onCreate.getTemplate": "fluid.tests.panels.utils.getTemplate"}
```

Comment:
This listener declaration can be more declarative:

listeners: {
    "onCreate.getTemplate": {
        funcName: "flud.fetchResources",
        args: ["{that}.options.resources", "{that}.refreshView"]
    }
}

If it works, you can get rid of "fluid.tests.panels.utils.injectTemplates.getTemplate".